lib/related_articles/database.rb

```diff
@@ -0,0 +1,118 @@
+# TODO: file an issue about weaviate-ruby relying on Object#present?
+class Object; def present?; !!self; end; end
```

Comment:
Possible issue to inquire about upstream with  `weaviate-ruby` – seems to rely on an ActiveSupport extension, which I've recreated with this little monkey-patch.